jvm/src/main/scala/io/glutenproject/expression/ConverterUtils.scala

```diff
@@ -250,18 +328,34 @@ object ConverterUtils extends Logging {
         case BooleanType =>
           // TODO: Not in Substrait yet.
           typedFuncName.concat("bool")
+        case ByteType =>
+          typedFuncName.concat("i8")
+        case ShortType =>
+          typedFuncName.concat("i8")
         case IntegerType =>
           typedFuncName.concat("i32")
         case LongType =>
           typedFuncName.concat("i64")
+        case FloatType =>
+          typedFuncName.concat("fp32")
         case DoubleType =>
           typedFuncName.concat("fp64")
         case DateType =>
           typedFuncName.concat("date")
+        case TimestampType =>
+          typedFuncName.concat("ts")
         case StringType =>
           typedFuncName.concat("str")
+        case BinaryType =>
+          typedFuncName.concat("bin")
```

Comment:
Please make the name be consistent with Substrait definitions at https://substrait.io/extensions/#function-signature-compound-names, for example, `bin` to `vbin`.

Reply:
fixed.